backends/redis.go

```diff
@@ -254,86 +182,24 @@ func (r *RedisBackend) GetAll(orgId string) []interface{} {
 		keys[i] = r.KeyPrefix + v
 	}
 
-	client := db
-	values := make([]interface{}, 0)
-
-	switch v := client.(type) {
-	case *redis.ClusterClient:
-		{
-			getCmds := make([]*redis.StringCmd, 0)
-			pipe := v.Pipeline()
-			for _, key := range keys {
-				getCmds = append(getCmds, pipe.Get(ctx, key))
-			}
-			_, err := pipe.Exec(ctx)
-			if err != nil && err != redis.Nil {
-				logger.Error("Error trying to get client keys: ", err)
-				return nil
-			}
-
-			for _, cmd := range getCmds {
-				values = append(values, cmd.Val())
-			}
-		}
-	case *redis.Client:
-		{
-			result, err := v.MGet(ctx, keys...).Result()
-			if err != nil {
-				logger.Error("Error trying to get client keys: ", err)
-				return nil
-			}
-
-			for _, val := range result {
-				values = append(values, val)
-			}
-		}
+	var values []interface{} = make([]interface{}, len(keys))
+	for i, s := range keys {
+		values[i] = s
 	}
-
 	return values
 }
 
 func (r *RedisBackend) cleanKey(keyName string) string {
 	return strings.Replace(keyName, r.KeyPrefix, "", 1)
 }
 
-func singleton(cache bool) redis.UniversalClient {
-	if cache {
-		v := singleCachePool.Load()
-		if v != nil {
-			return v.(redis.UniversalClient)
-		}
-		return nil
-	}
-	v := singlePool.Load()
-	if v != nil {
-		return v.(redis.UniversalClient)
-	}
-	return nil
+func (r *RedisBackend) hashKey(in string) string {
+	// missing implementation
+	return in
```

Comment:
are we going to implement this method or what's the plan? 🤔

Reply:
removed!